packages/sources/gsr/src/config/index.ts

```diff
@@ -27,4 +27,30 @@ export const config = new AdapterConfig({
     required: true,
     sensitive: true,
   },
+  LWBA_API_ENDPOINT: {
+    type: 'string',
+    description: 'The HTTP API endpoint to use to retrieve tokens for LWBA',
+    default: 'https://oracle.pre-prod.gsr.io/v1',
+  },
+  LWBA_WS_API_ENDPOINT: {
+    type: 'string',
+    description: 'The WS API endpoint to use for LWBA data',
+    default: 'wss://oracle.pre-prod.gsr.io/oracle',
+  },
```

Comment:
Ideally we use the `prod` endpoint if possible. If not, this is fine.

Reply:
agreed, waiting ~1 week or so for GSR to move their pre-prod data to the existing endpoint and for the go-ahead from DP